benchmarks/opmphm.c

```diff
@@ -734,7 +746,11 @@ static void benchmarkMappingAllSeeds (void)
 {
 	// create the n array
 	const size_t nCount = 7;
-	size_t n[nCount];
+	size_t * n = elektraMalloc (nCount * sizeof (size_t));
```

Comment:
So there was no way to have an array on the stack from a const size_t without warning?